rules/do-you-know-which-environments-you-need-to-provision-when-starting-a-new-project/rule.md

```diff
@@ -0,0 +1,49 @@
+---
+type: rule
+title: Do You Know Which Environments You Need To Provision When Starting A New
+  Project?
+uri: do-you-know-which-environments-you-need-to-provision-when-starting-a-new-project
+authors:
+  - title: Daniel Mackay
+    url: https://ssw.com.au/people/daniel-mackay/
+created: 2023-11-20T05:48:37.152Z
+guid: 8d165756-d35e-4d1a-8924-3c4f59523db4
+related:
+- ephemeral-environments
+- do-you-have-separate-development-testing-and-production-environments
+---
+Before any project can be used by a customer it must first be published to a production environment.  However, in order to provide a robust and uninterrupted service to customers, it is important that the production environment is not used for development or testing purposes.  To ensure this, we must setup a separate environment for each of these purposes.
+
+## What is each environment for?
+
+* **Production**: Real data being used by real customers
+* **Staging**: 'Production-like' environment used for final sign-off. Used for testing and verification before deploying to Production. Should be as close to Production as possible e.g. access (avoid giving - developers admin rights), same specs as production (especially during performance testing). However, this is not always the case due to cost implications.  It is important that staging is 'logically equivalent' to production.  This means having the same level or redundancy (e.g. Regions + Zones), back-ups, permissions, and service SLAs.
```

Comment:
is this a typo? "This means having the same level or redundancy" ??

I think it should read: "This means having the same level of redundancy"

Change from "or redundancy" 
To "of redundancy"

Reply:
Updated.